### PR TITLE
535-575 history

### PR DESCRIPTION
**Pull request by Abian36**

### CODE REVIEW

File: WTWSMS/common/dynasties/00_dynasties.txt [hunk 1]
```diff
@@ -32444,13 +32444,38 @@
 	culture = "italian"
 	religion = "catholic"
 }
+1051154 = {
+	name = "Solomonid"
+	culture = "greek"
+	religion = "nicene"
+}
```
Comment (loup99) [inline WTWSMS/common/dynasties/00_dynasties.txt]: Previously we decided against using "id" for Greek characters to instead use "ites", as you have done for the other dynasty names.

File: WTWSMS/common/dynasties/00_dynasties.txt [hunk 2]
```diff
@@ -32444,13 +32444,38 @@
...
 	culture = "visigothic"
 	religion = "arian"
-}
\ No newline at end of file
+}
+13100021= {
+	name = "Danielid" 
+	culture = "greek"
+	religion = "nicene"
+ }
```
Comment (loup99) [inline WTWSMS/common/dynasties/00_dynasties.txt]: Same as above.

**Comment by loup99** [comment]
What is WTWSMS/history/titles/titles - Acceso directo.lnk?

File: WTWSMS/common/dynasties/00_dynasties.txt (hunk 1 above)
Comment (Abian36) [reply WTWSMS/common/dynasties/00_dynasties.txt]: Oh yes! I forgot about it. I'll change it ASAP.

**Comment by Abian36** [comment]
I don't know when that titles - Acceso directo.lnk was created/uploaded but it wasn't intentional. I've deleted it.

File: WTWSMS/history/characters/greek.txt [hunk 3]
```diff
@@ -3802,6 +3827,18 @@
 		death=yes
 	}
 }
+13100018 = {
+    name="Callinicus" #PLRE vol.3 Ioannes 1
```
Comment (IhateTrains) [inline WTWSMS/history/characters/greek.txt]: Looks like an error in the comment.

File: WTWSMS/history/characters/greek.txt [hunk 4]
```diff
@@ -3825,4 +3862,17 @@
 	598.1.1={
 		death=yes
 	}
+}
+
+13100021={  #Danielus 4 PLRE VOL II
+   name= Danielus
```
Comment (IhateTrains) [inline WTWSMS/history/characters/greek.txt]: I'm not sure if it breaks something, but the name should be in quotation marks.

File: WTWSMS/history/characters/greek.txt [hunk 5]
```diff
@@ -2972,6 +2972,19 @@
 	}
 }
 
+1310016 = {
+	name="Solomon"
```
Comment (IhateTrains) [inline WTWSMS/history/characters/greek.txt]: Could you add a PLRE info comment, so it's easier to find him and avoid duplicates?

File: WTWSMS/history/characters/greek.txt (hunk 3 above)
Comment (Abian36) [reply WTWSMS/history/characters/greek.txt]: I can confirm, I thought about changing it long ago but I never remember doing it.

File: WTWSMS/history/characters/greek.txt (hunk 4 above)
Comment (Abian36) [reply WTWSMS/history/characters/greek.txt]: Ok, changing it

File: WTWSMS/history/characters/greek.txt (hunk 5 above)
Comment (Abian36) [reply WTWSMS/history/characters/greek.txt]: Sure

**Comment by Abian36** [comment]
Everything solved

File: WTWSMS/history/characters/greek.txt [hunk 6]
```diff
@@ -3778,6 +3791,18 @@
 		death=yes
 	}
 }
+1310017 = {
+    name="Iacobus " #PLRE vol.3 Iacobus 1
```
Comment (IhateTrains) [inline WTWSMS/history/characters/greek.txt]: needless space in name

File: WTWSMS/history/characters/roman.txt [hunk 7]
```diff
@@ -3480,6 +3480,18 @@
 		death=yes
 	}
 }
+1310020 = {
+    name="Flavius" #Flavius Ortalinus PLRE III Consul of Lydia
```
Comment (loup99) [inline WTWSMS/history/characters/roman.txt]: Isn't Ortalinus the name too?

Comment (Abian36) [reply WTWSMS/history/characters/roman.txt]: I´m not so sure about it, Ortalinus doesn´t sound as a roman name.

Comment (IhateTrains) [reply WTWSMS/history/characters/roman.txt]: Practically everything ending with -us sounds like a Roman name :P

Comment (Abian36) [reply WTWSMS/history/characters/roman.txt]: What should I do, invent a dynasty or leave it blank?

**Comment by loup99** [comment]
What happened to 6860?

**Comment by Abian36** [comment]
Who?

**Comment by Abian36** [comment]
Oh, Ok, I decided to replace him for the years the character I added really held the lands, as you told me to do with another character in the last pull request. He isn´t delted or anything so I can just readd him if you want.

File: WTWSMS/history/titles/c_belgrade.txt [hunk 8]
```diff
@@ -7,4 +7,7 @@
 }
 540.1.1={
 	holder=6831 
+}
+574.1.1={
+    holder=1310016
```
Comment (loup99) [inline WTWSMS/history/titles/c_belgrade.txt]: Singidunum is the city which became Belgrade, not Sirmium.

File: WTWSMS/history/characters/greek.txt [hunk 9]
```diff
@@ -3802,6 +3827,18 @@
...
+	religion="nicene"
+	culture="greek"
+	dynasties=1310001
+	499.1.1={
+		birth=yes
+	}
+	548.1.1={
+		death=yes
+	}
+}
```
Comment (loup99) [inline WTWSMS/history/characters/greek.txt]: This character should be a count, I think.

File: WTWSMS/history/titles/c_belgrade.txt (hunk 8 above)
Comment (Abian36) [reply WTWSMS/history/titles/c_belgrade.txt]: I know but it says that he was Duke of Sirmium (Which I decided to replace with the duchy of Moesia Superior, as IMO it is more optimal to give) and Belgrade is under him.

File: WTWSMS/history/characters/greek.txt (hunk 9 above)
Comment (Abian36) [reply WTWSMS/history/characters/greek.txt]: PLRE says he was governor of Cilicia, and actually fits with the time his predecessor left office.

Comment (loup99) [reply WTWSMS/history/characters/greek.txt]: Praeses of Ciliciae Secunda rather.

File: WTWSMS/history/titles/c_belgrade.txt (hunk 8 above)
Comment (loup99) [reply WTWSMS/history/titles/c_belgrade.txt]: So count rather?

Comment (Abian36) [reply WTWSMS/history/titles/c_belgrade.txt]: Maybe, but I gave him duchy title to share the land a bit more.

File: WTWSMS/history/characters/greek.txt (hunk 9 above)
Comment (IhateTrains) [reply WTWSMS/history/characters/greek.txt]: Cilicia Prima and Cilicia Secuda were separate provinces with Cilicia Prima being the more important one. I think @Abian36 confused both these provinces, treating governors of both provinces as just governors of Cilicia.
For reference, see the setup in Cilicia in the pre-476 history branch (some commits ago, though, as it's currently temporarily broken after the merge from master branch).

Comment (Abian36) [reply WTWSMS/history/characters/greek.txt]: The problem is that the mod only has one Cilicia.
Of course I´ll check how you have managed to solve it.

Comment (IhateTrains) [reply WTWSMS/history/characters/greek.txt]: The problem could be solved by at least giving the governors of Cilicia Secunda counties in the area of the province and making them vassals of k_cilicia, held by governors of Cilicia Prima.
In my branch I created the d_cilicia_secunda title, but it is not necessary in master branch if the team doesn't want to change the setup.

Comment (Abian36) [reply WTWSMS/history/characters/greek.txt]: I´ve seen what you have done and I think it is optimous but still not 100% sure. We´ll discuss this in the evening.

**Pull request merged**